auth/src/main/java/com/hms/lib/commonmobileservices/auth/google/GoogleAuthServiceImpl.kt

```diff
@@ -416,4 +416,25 @@ class GoogleAuthServiceImpl(private val context: Context) : AuthService {
 
         return work
     }
+
+    override fun linkWithPhone(
+        countryCode: String,
+        phoneNumber: String,
+        password: String,
+        verifyCode: String
+    ): Work<AuthUser> {
+        val work: Work<AuthUser> = Work()
+
+        val preferences = context.getSharedPreferences(CMS_SHARED_PREF, Context.MODE_PRIVATE)
+        val storedVerificationId = preferences.getString(VERIFICATION_ID, null)
+
+
+        val credential = PhoneAuthProvider.getCredential(storedVerificationId!!, verifyCode)
```

Comment:
Could you add null check for storedVerificationId value before credential process ?

Reply:
resolved.

Reply:
not resolved. Could u please add null check for storedVerificationId value before credential process ?